doc/userguide/upgrade.rst

```diff
@@ -50,6 +50,8 @@ Major changes
 - ``SIP_PORTS`` variable has been introduced in suricata.yaml
 - Application layer's ``sip`` counter has been split into ``sip_tcp`` and ``sip_udp``
   for the ``stats`` event.
+- Decoder and logger for ARP protocol has been introduced.
```

Comment:
can you also document the logger in the yaml docs and the eve record in the eve format docs?

Reply:
Right, sure.